R/dependencies.R

```diff
@@ -307,3 +307,271 @@ ALLSMALLER <- as.symbol("ALLSMALLER")
 #' @rdname selectors
 #' @export
 MATCH <- as.symbol("MATCH")
+
+
+# Dash 2 Syntax Functions
```

Comment:
Why are these present both in `dash_2.R` and here?

Reply:
Removed in 0446581.